cmd/telemeter-server/main.go

```diff
@@ -211,13 +191,21 @@ func (o *Options) Run() error {
 		}
 		authorizeURL = u
 
+		var transport http.RoundTripper
+
+		transport = &http.Transport{
+			Dial:                (&net.Dialer{Timeout: 10 * time.Second}).Dial,
```

Comment:
The `Dial` field has been deprecated since a few Golang releases ago in favor of `DialContext` [0]. We should clean this up in a follow up as well.

[0] https://golang.org/pkg/net/http/#Transport

Reply:
indeed, good catch, this needs a follow-up.